pkg/apic/apiserver/models/catalog/v1alpha1/Asset.go

```diff
@@ -40,7 +40,7 @@ type Asset struct {
 
 	Icon interface{} `json:"icon"`
 
-	Owner interface{} `json:"owner"`
+	Owner *apiv1.Owner `json:"owner"`
```

Comment:
Should these all have omitempty?

Reply:
I'm wondered the same thing. The first generated code that was blowing up when it wasn't a pointer was Secret.go. Turning it into a pointer fixed it, even without the omitempty. So, it seems like it is fine as is. I wasn't supplying an Owner to Secret, so it would have been empty.

Reply:
Can you check what that payload was?  Would it make sense to send this when the owner info is not sent?

`
{
 "name": "secret",
 "owner": {},
 "spec": ...
}
`

vs

`
{
 "name": "secret",
 "spec": ...
}
`

Reply:
I'm not sure why we would want to treat owner here any different than any of the other possible fields (e.g. attributes, tags, etc) that we don't pass if we don't have values for them.